about/index.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About Doodle Dandy  | Doodle Dandy</title>
```

Comment:
You have too many spaces here - there should be one space before and after the pipe character (`|`). Not sure if you want to say `About Doodle Dandy` or just `About us` since Doodle Dandy is already in the title?